packages/xo-server/src/api/vm.mjs

```diff
@@ -186,6 +186,8 @@ export const create = defer(async function ($defer, params) {
     }
   }
 
+  params.tags = resourceSet !== undefined ? (await this.getResourceSet(resourceSet)).tags : undefined
```

Comment:
`vm.create` accepts non-specific params that are passed as such to `vm.edit`, are you sure `tags` couldn't be used until now?

Reply:
From what I see it doesn't seem to be used, but in order to avoid conflicts I'm going to add a new "tags" parameter to the `vm.create` method.

Reply:
Either:
- accept the param but don't override it, merge new tags if necessary
- don't accept the param and make it obvious

The worst behavior would be discarding/overriding it silently.